string first_box_name
string second_box_name
---
string message_response